### PR TITLE
TIP-706: Add mapping configuration for family

### PR DESCRIPTION
**Pull request by SamirBoulil**
**ES Filter Checklist:**
- [X] Review the PQB filter integration tests
- [X] Add index mapping integration tests 
- [X] Write the filter & Spec
- [X] Update the documentation in `search_specification.rst`

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by anaelChardan — APPROVED** [review]

**Review by jjanvier — APPROVED** [review]
GTM if integration tests are green

no need to update the doc?

**Comment by SamirBoulil** [comment]
Yes you are right, I modified the mapping for this property. We need to make it appear in the search_specification.

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst [hunk 1]
```diff
@@ -899,53 +902,68 @@ Same as ``IN``, but with the ``must_not`` occured type
 
 Family
 ******
-:Apply: apply on the 'family' field
+:Apply: apply on datatype 'keyword' on the 'family' field
```
Comment (anaelChardan) [inline src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst]: `:Apply: apply 'keyword' datatype on 'family' field` would be better :)

Comment (SamirBoulil) [reply src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst]: yep

**Comment by SamirBoulil** [comment]
Build result is here https://core-ci.akeneo.com/blue/organizations/jenkins/jjanvier%2Fpim-community-dev/detail/PR-75/1/tests

**Pull request merged**